pkgs/by-name/ca/cargo-spellcheck/package.nix

```diff
@@ -21,7 +20,10 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ rustPlatform.bindgenHook ];
 
-  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ Security SystemConfiguration ];
+  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
+    darwin.apple_sdk.frameworks.Security
+    darwin.apple_sdk.frameworks.SystemConfiguration
+  ];
```

Comment:
You may want to also [migrate to the new Darwin SDK pattern](https://github.com/NixOS/nixpkgs/issues/354146)

Reply:
Thanks, done!

Reply:
It seems the recent force push was only a rebase. Let me know if you're still interested in making this change, otherwise ping me in a couple days for a merge if there are no other reviews :)

Reply:
Sorry if I am missing something :sweat_smile: I did a rebase and changed `darwin.apple_sdk.frameworks.` to `darwin.apple_sdk_11_0.frameworks.`, was there more to it than that?

Reply:
The new Darwin SDK actually removes the need for these individual frameworks entirely. This line should be able to become `buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ apple-sdk_11 ]` (the issue linked above also provides some other examples)

Reply:
Ah, gotcha, sorry, reading the original post it's clear now, my apologies :facepalm: 

Should be good to go now!